src/layouts/components/Footer.tsx

```diff
@@ -1,314 +1,260 @@
+        altLabel: "defichainwhitepaper",
+        isExternaLink: false,
+      },
+      {
+        label: "GitHub",
+        pathname: "/github",
+        testId: "github",
+        altLabel: "github",
+        isExternaLink: true,
+        url: "https://github.com/DeFiCh/ain",
+        hasUpArrowIcon: true,
+      },
+      {
+        label: "Developer Resource",
+        pathname: "/developerResource",
+        testId: "developerResource",
+        altLabel: "developerResource",
+        isExternaLink: false,
+      },
+      {
+        label: "Media Assets",
+        pathname: "/mediaAssets",
+        testId: "mediaAssets",
+        altLabel: "mediaAssets",
+        isExternaLink: false,
+      },
+      {
+        label: "Security",
+        pathname: "/security",
+        testId: "security",
+        altLabel: "security",
+        isExternaLink: false,
+      },
+    ],
+  },
+  {
+    category: "COMMUNITY",
+    childLink: [
+      {
+        label: "Blog",
+        pathname: "/blog",
+        testId: "blog",
+        altLabel: "blog",
+        isExternaLink: false,
+      },
+      {
+        label: "Newsletter",
+        pathname: "/newsletter",
+        testId: "newsletter",
+        altLabel: "newsletter",
+        isExternaLink: false,
+      },
+    ],
+  },
+];
 
-export function Footer(): JSX.Element {
-  const { t } = useTranslation("layout");
+export default function Footer(): JSX.Element {
+  const router = useRouter();
 
+  // default set language to be english to be directed to international telegram link
+  const [language, setLanguage] = useState(router.locale ?? "");
+
+  // eslint-disable-next-line consistent-return
+  useEffect(() => {
+    switch (router.locale) {
+      case "de":
+        return setLanguage("_DE");
+      case "fr":
+        return setLanguage("_FR");
+      default:
+        setLanguage("");
+    }
+  }, [router]);
   return (
-    <footer className="mt-20 mb-12">
-      <Container>
-        <div className="flex flex-col">
-          <Link
-            href={{ pathname: "/" }}
-            passHref
-            className="flex items-center cursor-pointer hover:text-primary-500"
-            data-testid="Footer.Logo"
-          >
-            <DeFiChainLogo className="w-40 lg:w-52 mb-2" />
-          </Link>
-          <h2
-            className="text-xl lg:text-2xl font-semibold"
-            data-testid="Footer.Heading"
-          >
-            {t("footer.title")}
-          </h2>
-        </div>
-        <div className="mt-10 flex flex-wrap space-y-8 xl:space-y-0">
-          <div className="flex flex-wrap max-w-sm lg:max-w-max w-full lg:w-2/3 xl:w-1/2 space-y-8 lg:space-y-0">
-            <div className="lg:w-1/2">
-              <FooterSectionSitemap />
-            </div>
-            <div className="lg:w-1/2">
-              <FooterSectionSocial />
-            </div>
+    <footer className="relative bg-dark-00">
+      <Container className="relative z-10 py-10 px-0 md:px-10 lg:px-0">
```

Comment:
for 1440px screens, padding should be `px-[120px]` and `144px` to the bottom

Reply:
for 768px & 375px screen, `mx-auto` from Container is making the width weird

Reply:
Removed initial props from original Container